packages/overlay/src/integrations/sentry/components/events/error/Frame.tsx

```diff
@@ -48,51 +70,44 @@ export default function Frame({
 }) {
   const [isOpen, setOpen] = useState(defaultExpand);
 
-  const hasSource = !!frame.context_line;
+  const hasSource = Boolean(frame.context_line);
   const fileName = platform === 'java' ? frame.module : frame.filename || frame.module;
   return (
-    <li className="border-primary-900 bg-primary-900 border-b last:border-b-0">
+    <li
+      className={classNames(
+        hasSource ? 'cursor-pointer' : '',
+        !isOpen && hasSource ? 'hover:bg-primary-900' : '',
+        'bg-primary-950 border-primary-900 my-1 overflow-hidden rounded-md border',
+      )}
+    >
       <div
         className={classNames(
-          hasSource ? 'hover:bg-primary-800 cursor-pointer' : '',
-          'border-primary-950 text-primary-400 border-b px-2 py-1',
+          'text-primary-400 flex items-center justify-between px-2 py-1',
+          isOpen ? 'bg-primary-900' : '',
         )}
         onClick={hasSource ? () => setOpen(!isOpen) : undefined}
       >
-        {fileName ? (
-          <span className="text-primary-100">
-            {formatFilename(fileName)}
-            {' in '}
-          </span>
-        ) : null}
-
-        <span className="text-primary-100">{frame.function}</span>
-        {frame.lineno !== undefined && (
-          <>
-            {' '}
-            at line{' '}
+        <div>
+          {fileName ? (
             <span className="text-primary-100">
-              {frame.lineno}
-              {frame.colno !== undefined && `:${frame.colno}`}
+              {formatFilename(fileName)}
+              {' in '}
             </span>
-          </>
-        )}
-        {isOpen && !frame.filename?.includes(':') ? (
```

Comment:
looks like we missed this condition `!frame.filename?.includes(':')`

Reply:
This was intentional but we can revisit the decision in the future if necessary.